release-notes/unreleased/2043.feat.rst

```diff
@@ -0,0 +1,9 @@
+Added a new transpiler translation plugin :class:`IBMFractionalTranslationPlugin` 
+and a pass :class:`FoldRzzAngle`.
+This plugin is automatically applied for backends
+retrieved with the `use_fractional_gates` optin,
+and the folding pass is added when the backend target includes the `RZZ` gate.
+
+The new pass modifies the input quantum circuit, so that all `RZZ` gates in the
+circuit have an angle parameter within [0, pi/2] which is supported 
+by IBM Quantum processors.
```

Comment:
```suggestion
Added a new transpiler translation plugin :class:`~.IBMFractionalTranslationPlugin` 
and a pass :class:`~.FoldRzzAngle`.
This plugin is automatically applied for backends
retrieved with the ``use_fractional_gates`` opt-in,
and the folding pass is added when the backend target includes the ``RZZ`` gate.

The new pass modifies the input quantum circuit, so that all ``RZZ`` gates in the
circuit have an angle parameter within [0, pi/2] which is supported 
by IBM Quantum processors.
```